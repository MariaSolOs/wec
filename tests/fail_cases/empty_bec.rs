use wec::bec;

// Make sure the behavior of bec![] is the same as vec![] when creating empty
// vectors.

fn main() {
    let v = bec![];
}
