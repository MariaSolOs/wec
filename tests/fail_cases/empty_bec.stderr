error[E0282]: type annotations needed for `Vec<_>`
 --> tests/fail_cases/empty_bec.rs:7:9
  |
7 |     let v = bec![];
  |         ^
  |
help: consider giving `v` an explicit type, where the placeholders `_` are specified
  |
7 |     let v: Vec<_> = bec![];
  |          ++++++++
